update the readme (#110)
dart-lang/web: README.md

Lightweight browser API bindings built around JS static interop.

## What's this?

> This package exposes the browser's DOM APIs. It's generated from the Web IDL

Reviewer: ```suggestion
This package exposes browser APIs. It's generated from the Web IDL
```

We expose more than just the DOM APIs.

Reviewer 2: true true!

definitions and uses recent Dart language features for zero-overhead bindings.

This package is intended to replace
[`dart:html`](https://api.dart.dev/stable/dart-html/dart-html-library.html) and

> similar Dart SDK libraries. The package will support DOM access from Dart code

Reviewer 2: ```suggestion
similar Dart SDK libraries. It will support DOM access from Dart code
```

compiled to either JavaScript or WebAssembly.

## Status

The APIs in this package are still fairly provisional and could change rapidly.

> We'll continue to use semver as we evolve the APIs.

Reviewer 2: Maybe `Semantic Versioning` with a link to the spec?

Author: 👍  - done
